server/index.test.js

```diff
@@ -0,0 +1,81 @@
+const rp = require('request-promise');
+const db = require('./db.js');
+const Product = require('./model.js');
+const { app } = require('./index.js');
+
+
+// `rpOptionsGet` and `rpOptionsPost` generate options objects for rp.
+const rpOptionsGet = (endpoint) => {
+  return {
+    uri: `http://localhost:3001${endpoint}`,
+    json: true
+  };
+};
+
+const rpOptionsPost = (id, bid) => {
+  return {
+    method: 'POST',
+    url: `http://localhost:3001/bid/${id}`,
+    body: {bid},
+    json: true
+  };
+};
+
+// The database should be seeded with `npm run seed` before these tests are run.
```

Comment:
Would these instructions useful in the README?

Reply:
Actually, now that I think about it. This is for testing. Testing happens in development. No need to add to README

Reply:
Hmm, thanks. It might be worth making a note about it at some point.